README.md

```diff
@@ -116,6 +115,8 @@ I would like to thank Microverse for giving me all the necessary knowledge I nee
 
 <!-- LICENSE -->
 
+## 📝 License <a name="license"></a>
+
 This project is [MIT](./MIT.md) licensed.
 
```

Comment:
![image](https://github.com/Bhekisile/School-Library/assets/93039359/c713c4ba-0497-4a95-acf6-df3a742de21b)


**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] Great work on adding a license file but i guess your license file is not properly setup 😨 Kindly replace the current year instead of 2021 and Database App instead of APP TYPE 🤔 to make your readme descriptive and professional 👍

Reply:
It's fixed. Thank you